microsite/data/plugins/stack-overflow.yaml

```diff
@@ -0,0 +1,9 @@
+---
+title: Stack Overflow
+author: Spotify
+authorUrl: https://github.com/spotify
+category: Discovery
+description: Provides Stack Overflow specific functionality that can be used in different ways (e.g. for homepage and search) to compose your Backstage App.
+documentation: https://github.com/backstage/backstage/blob/master/plugins/stack-overflow
+iconUrl: img/stack-overflow-logo.svg
```

Comment:
Feeling that https://stackoverflow.com/legal/trademark-guidance doesn't give us a clear enough permission to do this tbh. Perhaps either check with them if you haven't already, or if we could link directly to an external resource?

Reply:
Hey, I've checked with them (mail convo) and the logo was sent over by them and its all approved. Do we need some written approval for it too?

Reply:
Sounds good enough! 👍